samples/spring-valueentity-counter/src/main/java/com/example/action/Confirmed.java

```diff
@@ -1,5 +1,8 @@
 package com.example.action;
 
+import com.fasterxml.jackson.annotation.JsonTypeInfo;
+
+@JsonTypeInfo(use = JsonTypeInfo.Id.NAME, property = "type")
 public class Confirmed {
```

Comment:
This has little to do with this PR. It was failing the serialization when running the integration test in this sample. Although the tests were passing.